src/main/webapp/scripts/ou-script.js

```diff
@@ -1,18 +1,222 @@
 var params = JSON.parse(localStorage.getItem('oauth2-test-params'));
 var token = params['access_token'];
 
-function getAllOUs(){
-    console.log(token);
+/*
+ * Onload: fetches all OUs, constructs parent-child hierarchy JSON, creates tree visualization.
+*/
+function getAllOUs() {
+    // console.log(token);
+    // fetch all OUs from API
     fetch('https://www.googleapis.com/admin/directory/v1/customer/my_customer/orgunits?orgUnitPath=/&type=all', {
     headers: {
         'authorization': `Bearer ` + token,
     }
     }).
     then(response => response.json())
-    .then((ous) => {
-        console.log(ous);
+    .then(directoryOUs => directoryOUs['organizationUnits'])
+    .then((orgUnits) => {
+        orgUnits.sort(pathDepthSort); // sort by file depth
```

Comment:
should it be "OU depth" or "Path depth" instead of "file depth" give that we are working with OU structures

Reply:
yep, sounds like a better way to word it. will update that in a future PR!